Sources/Purchasing/EntitlementInfo.swift

```diff
@@ -40,6 +40,9 @@ import Foundation
     /// For entitlements granted via the Amazon Store.
     @objc(RCAmazon) case amazon = 6
 
+    /// For entitlements granted via RC Billing
+    @objc(RCRCBilling) case rcBilling = 7
```

Comment:
I wasn't really sure about the ObjC name, but I didn't want to call it `RCBilling` thoughts?

Reply:
Hmm well... I guess as long as we don't have any other `Billing` stores in the future, it should be ok 😅. Will rename it.

Reply:
Ha uh... I do agree that `RCBilling` is _probably_ the right way to go even though it feels weird because different pattern 🤷‍♂️